books/models.py

```diff
@@ -871,6 +869,10 @@ def subjects(self):
             subject_list.append(subject.subject_name)
         return subject_list
 
+    def errata_content(self):
```

Comment:
I think you want to add `@property` to the top of this. (like `book_title` above). Then add this to the `api_fields`

Reply:
The API was working without @property, so I didn't add it. It is there now.

Reply:
Adding `@property` allows you to treat this like another field, instead of a method.
So for example, you can call `book.errata_content` instead of `book.errata_content()`.

Just makes it a little clearer to work with.